### PR TITLE
10.0 [MIG] stock_scanner_inventory

### PR DESCRIPTION
**Pull request by misern2**
Standard migration.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
You can now adapt this migration (and the rest of PRs) to the new menu on parent one.

**Comment by pedrobaeza** [comment]
It seems there's an incorrect rebase

**Comment by misern2** [comment]
@pedrobaeza rebase fixed. Could you please check if it's ok now ?

File: stock_scanner_inventory/i18n/stock_scanner_inventory.pot [hunk 1]
```diff
@@ -1,81 +0,0 @@
-# Translation of Odoo Server.
```
Comment (pedrobaeza) [inline stock_scanner_inventory/i18n/stock_scanner_inventory.pot]: Don't remove the POT file. Due to the new translation platform, is now needed. It's going to be auto-generated, but removing it, this makes the diff larger.

File: stock_scanner_inventory/README.rst [hunk 2]
```diff
@@ -1,65 +1 @@
...
-   :target: https://odoo-community.org
-
-This module is maintained by the OCA.
-
-OCA, or the Odoo Community Association, is a nonprofit organization whose
-mission is to support the collaborative development of Odoo features and
-promote its widespread use.
-
-To contribute to this module, please visit https://odoo-community.org.
+**This file is going to be generated by oca-gen-addon-readme.**
```
Comment (pedrobaeza) [inline stock_scanner_inventory/README.rst]: Maybe you can give a try to install the README generator for having a smaller diff: one removing it and another (automatic) adding it.

**Comment by misern2** [comment]
Thanks for the comments. 

Changes done!

**Comment by pedrobaeza** [comment]
Merging optimistically as CIs and coverage are green

**Comment by rousseldenis** [comment]
@pedrobaeza You can do ping to review. You missed openerp => odoo

We are in the same room :smile:

**Comment by pedrobaeza** [comment]
Sorry, I used the optimistically merging criteria as this was stuck for a long time, but I didn't remember you are tracking all logistics parts.

**Comment by rousseldenis** [comment]
Arf, I think 'optimistic' should be used for little fixes or trivial PR's, not for whole module merge, no ?

**Pull request merged**